workshops/util.py

```diff
@@ -476,113 +476,113 @@ def generate_url_to_event_index(website_url):
     raise WrongWorkshopURL()
 
 
-def find_tags_on_event_index(content):
-    """Given workshop's raw `index.html`, find and take YAML tags that
+def find_metadata_on_event_index(content):
+    """Given workshop's raw `index.html`, find and take YAML metadata that
     have workshop-related data."""
     try:
         first, header, last = content.split('---')
-        tags = yaml.load(header.strip())
+        metadata = yaml.load(header.strip())
 
-        # get tags to the form returned by `find_tags_on_event_website`
+        # get metadata to the form returned by `find_metadata_on_event_website`
         # because YAML tries to interpret values from index's header
-        filtered_tags = {key: value for key, value in tags.items()
-                         if key in ALLOWED_TAG_NAMES}
-        for key, value in filtered_tags.items():
+        filtered_metadata = {key: value for key, value in metadata.items()
+                         if key in ALLOWED_METADATA_NAMES}
+        for key, value in filtered_metadata.items():
             if isinstance(value, int):
-                filtered_tags[key] = str(value)
+                filtered_metadata[key] = str(value)
             elif isinstance(value, datetime.date):
-                filtered_tags[key] = '{:%Y-%m-%d}'.format(value)
+                filtered_metadata[key] = '{:%Y-%m-%d}'.format(value)
             elif isinstance(value, list):
-                filtered_tags[key] = ', '.join(value)
+                filtered_metadata[key] = ', '.join(value)
 
-        return filtered_tags
+        return filtered_metadata
 
     except (ValueError, yaml.scanner.ScannerError):
         # can't unpack or header is not YML format
         return dict()
 
 
-def find_tags_on_event_website(content):
-    """Given website content, find and take <meta> tags that have
+def find_metadata_on_event_website(content):
```

Comment:
`find_metadata_on_event_index/homepage` extracts it from the YAML metadata that is at the beginning of raw `index.html` file while `find_metadata_on_event_website` extracts it from `html  > head > meta` tags